SystemReference/Gnc/Imu/Imu.hpp

```diff
@@ -19,7 +19,16 @@ class Imu : public ImuComponentBase {
 
     // Values for the static consts were attained from the data sheet of the
     // MPU6050
-    static const U8 I2C_DEV0_ADDR = 0x68;
+
+    //! The I2C device addresses
+    struct I2cDevAddr {
```

Comment:
We really ought to call this `ImuDevAddr`.  The I2C protocol is clear addresses at `U8`, however; it is the IMU that specifies the choice of 2 addresses.  I do like the revised structure!

Reply:
The full name of the type is `Gnc::Imu::I2cDevAddr`. Does this address the concern?